frontend/package.json

```diff
@@ -39,7 +39,7 @@
     "axios": "^0.23.0",
     "bootstrap": "^4.6.0",
     "bootstrap-vue": "^2.21.2",
-    "chart.js": "^2.9.4",
+    "chart.js": "2.9.4",
```

Comment:
Why did you remove hat?

Reply:
In order to install 2.9.4, the updated version won't work with our configuration